api/src/main/java/org/apache/iceberg/actions/ActionsProvider.java

```diff
@@ -70,4 +70,10 @@ default RewritePositionDeleteFiles rewritePositionDeletes(Table table) {
     throw new UnsupportedOperationException(
         this.getClass().getName() + " does not implement rewritePositionDeletes");
   }
+
+  /** Instantiates an action to compute table stats. */
+  default ComputeTableStats computeTableStats(Table table) {
+    throw new UnsupportedOperationException(
+        this.getClass().getName() + " does not implement ComputeTableStats");
```

Comment:
Minor: `ComputeTableStats` -> `computeTableStats` (we refer to the method name in this case, not class).